test/test_invalid_headers.py

```diff
@@ -300,6 +311,13 @@ def test_matching_authority_host_headers(self,
         assert headers == h2.utilities.validate_headers(
             headers, hdr_validation_flags)
 
+    @pytest.mark.parametrize('hdr_validation_flags',
+                             hdr_validation_response_headers)
```

Comment:
This test will presumably fail: some configurations of the header validation flags *allow* for no :status header field to be present because they are for non-response header blocks.